core/src/Streamly/Internal/Data/Array/Generic/Mut/Type.hs

```diff
@@ -27,6 +27,11 @@ module Streamly.Internal.Data.Array.Generic.Mut.Type
     , writeN
     , writeWith
     , write
+    , fromStreamN
+    , fromStream
+
+    , fromListN
+    , fromList
```

Comment:
What is the reason for exposing these? I do need see these used or exported from any exposed module. Are these supposed to be exposed from the released module?